app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,40 @@
+class OrdersController < ApplicationController
+  before_action :authenticate_employee!
+
+  def index
+    @orders = Order.where(employee_id: current_employee.id).includes(:reward)
+  end
+
+  def show
+    order
+  end
+
+  def new
+    @rewards = Reward.all
+  end
+
+  def create
+    @order = Order.new(**order_params, employee_id: current_employee.id)
```

Comment:
Why do we need the double splat operator here? 

```suggestion
    @order = Order.new(**order_params, employee: current_employee)
```